pdfminer/encodingdb.py

```diff
@@ -32,7 +33,7 @@ def name2unicode(name):
 
     else:
         if name in glyphname2unicode:
-            return glyphname2unicode.get(name)
+            return glyphname2unicode[name]
```

Comment:
Are you sure that returning `None` is not an option here?

Reply:
The line immediately above is `if name in glyphname2unicode:`, so we know it won't be None.